actions/databases.go

```diff
@@ -294,6 +300,33 @@ func DatabasesPut(c buffalo.Context) error {
 		log.Println("Modified RDS instance", instanceOutput)
 	}
 
+	if input.Tags != nil {
+		log.Println("Updating tags for "+c.Param("db")+"\n", input.Tags)
+		tags := &rds.AddTagsToResourceInput{}
+		tags.Tags = input.Tags
+
+		// determine ARN(s) for this RDS resource
+		arns, arnErr := rdsClient.DetermineArn(aws.String(c.Param("db")))
+		if arnErr != nil {
```

Comment:
i think its okay to shadow `err` here, since you're returning on the error

Reply:
just do the `aws.String` inside the function and pass a string to `rdsClient.DetermineArn`

Reply:
(commenting on this here even though I noticed it below)

I think it makes more sense for `rdsClient.DetermineArn` to have the signature

```golang
func (cl Client) DetermineArn(db string) ([]*string, error)
```

Reply:
I was debating which way was better 😄